### PR TITLE
Syntax: Variable

### LINKED ISSUE #3: [Syntax] Variable
"huh" automatically infers the characteristics of a variable based on the naming convention.

You can define and use the variable in the form of "the <variable name>".

For example, a variable written in camel-case will automatically have mutable characteristics.

### PR DESCRIPTION
**Pull request by rlaope**
# Variable

"huh" automatically infers the characteristics of a variable based on the naming convention.

You can define and use the variable in the form of "the <variable name>".

For example, a variable written in camel-case will automatically have mutable characteristics.

### Example

```
the userNickname = "USER"
userNickname = "USER1"

the UserNickname = "USER"
// UserNickname = "USER1" error: Variables written in Pascal case cannot be reassigned.

the user_name = "USER"
user_name = "USER1"
```

<br>

## Convention inference

Convention inference is designed as follows, and certain characteristics are determined when variables are created using the following convention.

convention | characteristic
---|---
camelCase | Variable
PascalCase | Constant
snake_case | Varialbe
Screaming_Snake_Case | Constant
UPPERCASE | Constant
kebab-case | Variable

### CODE REVIEW

File: sources/variable/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+"huh" automatically infers the characteristics of a variable based on the naming convention.
+
+You can define and use the variable in the form of "the <variable name>".
+
+For example, a variable written in camel-case will automatically have mutable characteristics.
+
+### Example
+
+```
```
Comment (onee-only) [inline sources/variable/README.md]:
```suggestion
```java
```

**Review by onee-only — COMMENTED** [review]
Wow, you are genius @esperar. 
Please take a look at my suggestions.
Otherwise, LGTM!

File: sources/variable/README.md (hunk 1 above)
Comment (rlaope) [reply sources/variable/README.md]: LGTM. Thanks!

**Review by onee-only — APPROVED** [review]
Good work!

**Pull request merged**